browser-test/src/applicant/navigation/northstar_application_navigation_with_eligibility_conditions.test.ts

```diff
@@ -0,0 +1,485 @@
+          fullProgramName,
+          'second description',
+          ['nav-predicate-email-q'],
+        )
+
+        await adminPrograms.gotoAdminProgramsPage()
+        await adminPrograms.publishProgram(fullProgramName)
+        await logout(page)
+      },
+    )
+
+    test('does not show Not Eligible when there is no answer', async ({
+      applicantQuestions,
+    }) => {
+      await applicantQuestions.clickApplyProgramButton(fullProgramName)
+      await applicantQuestions.clickReview(/* northStarEnabled= */ true)
+      await applicantQuestions.expectMayNotBeEligibleAlertToBeHidden()
+    })
+
+    test('shows not eligible with ineligible answer', async ({
+      page,
+      applicantQuestions,
+    }) => {
+      await applicantQuestions.applyProgram(
+        fullProgramName,
+        /* northStarEnabled= */ true,
+      )
+
+      await test.step('fill out application and submit', async () => {
+        await applicantQuestions.answerNumberQuestion('1')
+        await applicantQuestions.clickContinue()
+        await applicantQuestions.expectIneligiblePage(/* northStar= */ true)
+      })
+
+      // Verify the question is marked ineligible.
+      await test.step('Verify application is marked ineligible', async () => {
+        await applicantQuestions.gotoApplicantHomePage()
+        await applicantQuestions.seeEligibilityTag(
+          fullProgramName,
+          /* isEligible= */ false,
+        )
+      })
+
+      await test.step('verify not eligible alert is shown on review page', async () => {
+        await applicantQuestions.clickApplyProgramButton(fullProgramName)
+        await applicantQuestions.clickReview(/* northStarEnabled= */ true)
+
+        await applicantQuestions.expectMayNotBeEligibileAlertToBeVisible()
+        await applicantQuestions.expectIneligibleQuestionInReviewPageAlert(
+          AdminQuestions.NUMBER_QUESTION_TEXT,
+        )
+        await validateScreenshot(
+          page,
+          'application-ineligible-review-page',
+          /* fullPage= */ true,
+          /* mobileScreenshot= */ true,
+        )
+        await validateAccessibility(page)
+      })
+    })
+
+    test('shows may be eligible with an eligible answer', async ({
+      page,
+      applicantQuestions,
+    }) => {
+      await applicantQuestions.applyProgram(
+        fullProgramName,
+        /* northStarEnabled= */ true,
+      )
+
+      await test.step('fill out application without submitting and verify message on review page', async () => {
+        await applicantQuestions.answerNumberQuestion('5')
+        await applicantQuestions.clickReview(/* northStarEnabled= */ true)
+        await applicantQuestions.expectMayBeEligibileAlertToBeVisible()
+        await validateScreenshot(
+          page,
+          'application-eligible-review-page',
+          /* fullPage= */ true,
+          /* mobileScreenshot= */ true,
+        )
+      })
+
+      await test.step('verify tag on home page', async () => {
```

Comment:
**suggestion (non-blocking):** This would be a good time to start using "applicant home page" as the page name, following our new glossary.  :-D

Reply:
I'll update in #9279